notify/opsgenie/opsgenie.go

```diff
@@ -180,6 +204,55 @@ func (n *Notifier) createRequest(ctx context.Context, as ...*types.Alert) (*http
 			Note:        tmpl(n.conf.Note),
 			Priority:    tmpl(n.conf.Priority),
 		}
+		var buf bytes.Buffer
+		if err := json.NewEncoder(&buf).Encode(msg); err != nil {
+			return nil, false, err
+		}
+		req, err := http.NewRequest("POST", createEndpointURL.String(), &buf)
+		if err != nil {
+			return nil, true, err
```

Comment:
Can we detect here if an alert has been created or just grouped?

Reply:
unfortunately not, OpsGenie just returns 202 and processes the alert asynchronously

Reply:
@roidelapluie can I resolve this thread?